### ISSUE TITLE
Screen reader reads out commas on RadioSchedule

### ISSUE THREAD

**Issue by nicola-d-bbc** [bbc/simorgh#9593]
**Describe the bug**
When navigating the Radio Schedule component with a screen reader, visually hidden commas are read out. Similar to this bug: https://github.com/bbc/simorgh/issues/9559

Found this bug on TalkBack but all supported screen readers should be tested.

Additional example: https://www.bbc.com/persian/bbc_persian_radio/programmes/p0340vyx

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to 'https://www.bbc.com/hausa'
2. Navigate to 'Shirye-shiryenmu' heading using a screen reader
3. Swipe to the second card (Bug doesn't happen on cards with 'NA GABA/NEXT')
4. Swipe through the card's info
5. Two commas will be read out by the screen reader

**Expected behaviour**
Screen readers should pause instead of reading out 'comma'

**Screenshots**
![image](https://user-images.githubusercontent.com/52782713/138910783-5ed36f72-e03b-441f-a020-abcaeac38626.png)
![image](https://user-images.githubusercontent.com/52782713/138910972-5896f73d-cae7-446c-abc2-4d0f15391345.png)

**Testing notes**

- [x] This bug fix is expected to need manual testing.

**Checklist**
- [x] (BBC contributors only) This issue follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
